Panic on create order page

To reproduce:

Start the app, Go to settings and turn off exchange APIs. Go to the trade tab, Click on the Exchange btn that opens up the create order page. You will see a message that says to enable the exchange API to trade. Go back to settings and enable the Exchange APIs. After clicking the back btn on the settings page the create order page will be displayed and the message overlay will be removed. Selecting an asset will crash the app.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x100dd161d]

goroutine 13 [running]:
github.com/crypto-power/cryptopower/ui/page/components.(*WalletAndAccountSelector).SetSelectedAsset(0x0?, {0xc004d9e740?, 0x1?, 0xc0086b8e50?})
        /Users/philemon/Desktop/osp/cryptopower/ui/page/components/wallet_account_selector.go:145 +0x1d
github.com/crypto-power/cryptopower/ui/page/exchange.(*CreateOrderPage).updateAssetSelection(0xc000270000, {0xc00064bd10, 0xc0086b8e01?, 0xc005970a80?}, {0x0, 0x0, 0x100cac9e6?})
        /Users/philemon/Desktop/osp/cryptopower/ui/page/exchange/create_order_page.go:463 +0xef
github.com/crypto-power/cryptopower/ui/page/exchange.(*CreateOrderPage).updateWalletAndAccountSelector(0xc002c89040?, {0xc00064bd10?, 0x10004acb3?, 0x10003b036?}, {0x0?, 0xc00328f068?, 0x100cac9c0?})
        /Users/philemon/Desktop/osp/cryptopower/ui/page/exchange/create_order_page.go:210 +0x28
github.com/crypto-power/cryptopower/ui/page/exchange.NewCreateOrderPage.func1(0xc005972040)
        /Users/philemon/Desktop/osp/cryptopower/ui/page/exchange/create_order_page.go:171 +0x116
github.com/crypto-power/cryptopower/ui/page/components.NewAssetTypeSelector.func1(0xc00330acc0?)
        /Users/philemon/Desktop/osp/cryptopower/ui/page/components/asset_selector.go:68 +0x9d
github.com/crypto-power/cryptopower/ui/page/components.(*assetTypeModal).Handle(0xc005970a80)
        /Users/philemon/Desktop/osp/cryptopower/ui/page/components/asset_selector.go:234 +0x127
github.com/crypto-power/cryptopower/ui.(*Window).handleFrameEvent(0xc000392140, {{0xc142b3a5241b0ea8, 0xd472f80a4, 0x102758220}, {0x40000000, 0x40000000}, {0x640, 0x514}, {0x0, 0x0, ...}, ...})
        /Users/philemon/Desktop/osp/cryptopower/ui/window.go:235 +0x171
github.com/crypto-power/cryptopower/ui.(*Window).HandleEvents(0xc000392140)
        /Users/philemon/Desktop/osp/cryptopower/ui/window.go:206 +0x2b9
main.main.func3()
        /Users/philemon/Desktop/osp/cryptopower/main.go:110 +0x1d
created by main.main
        /Users/philemon/Desktop/osp/cryptopower/main.go:108 +0x54e

```

--- a/ui/page/exchange/create_order_page.go
+++ b/ui/page/exchange/create_order_page.go
@@ -60,6 +60,7 @@ type CreateOrderPage struct {
 	amountErrorText  string
 	fetchingRate     bool
 	rateError        bool
+	inited           bool
 
 	materialLoader material.LoaderStyle
 
@@ -219,11 +220,18 @@ func (pg *CreateOrderPage) OnNavigatedTo() {
 	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
 
 	if pg.isExchangeAPIAllowed() && pg.isMultipleAssetTypeWalletAvailable() {
-		pg.scheduler.SetChecked(pg.WL.AssetsManager.IsOrderSchedulerRunning())
-		pg.listenForSyncNotifications()
-		pg.loadOrderConfig()
-		go pg.scroll.FetchScrollData(false, pg.ParentWindow())
+		pg.initPage()
 	}
+}
+
+// initPage initializes required data on this page and should be called only
+// once after it has been displayed.
+func (pg *CreateOrderPage) initPage() {
+	pg.inited = true
+	pg.scheduler.SetChecked(pg.WL.AssetsManager.IsOrderSchedulerRunning())
+	pg.listenForSyncNotifications()
+	pg.loadOrderConfig()
+	go pg.scroll.FetchScrollData(false, pg.ParentWindow())
 }
 
 func (pg *CreateOrderPage) OnNavigatedFrom() {
@@ -594,7 +602,6 @@ func (pg *CreateOrderPage) Layout(gtx C) D {
 	var navBtn *cryptomaterial.Button
 
 	isTestNet := pg.Load.WL.AssetsManager.NetType() != libutils.Mainnet
-
 	switch {
 	case isTestNet:
 		msg = values.String(values.StrNoExchangeOnTestnet)
@@ -608,6 +615,10 @@ func (pg *CreateOrderPage) Layout(gtx C) D {
 	case !pg.isMultipleAssetTypeWalletAvailable():
 		msg = pg.errMsg
 		overlaySet = true
+	}
+
+	if !overlaySet && !pg.inited {
+		pg.initPage()
 	}
 
 	pg.scroll.OnScrollChangeListener(pg.ParentWindow())

